style.css

```diff
@@ -0,0 +1,632 @@
+@import url('https://fonts.cdnfonts.com/css/inter');
+
+.first {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header-first {
+  display: none;
+}
+
+.second-text {
+  font-size: 20px;
+}
+
+.second-text1 {
+  font-size: 40px;
+}
+
+.second {
+  background-image: url(./images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  vertical-align: top;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  padding: 10px 15px 15px 10px;
+  font-family: 'Inter', sans-serif;
+}
+
+.third {
+  font-style: normal;
+  font-weight: 400;
+  text-align: center;
+  line-height: 28px;
+  font-size: 20px;
+  vertical-align: top;
+  color: #172b4d;
+}
+
+.socials {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: flex-end;
+  list-style: none;
+  padding: 5px;
+  gap: 20px;
+}
+
+.green {
+  color: #36b37e;
+  animation-duration: 3s;
+  animation-name: slidein;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+}
+
+p {
+  animation-duration: 3s;
+  animation-name: slidein;
+}
```

Comment:
- [ ] [ optional ] This animation is fine but I think t should have a timeout, loading it continuously makes your page slow and not goog for the user experience